src/world_diff.rs

```diff
@@ -369,11 +370,14 @@ mod tests {
             first_changes in arbitrary_storage_changes(),
             second_changes in arbitrary_storage_changes(),
         ) {
-            let mut world_diff = WorldDiff::default();
-            world_diff.storage_initial_values = initial_values
+            let storage_initial_values = initial_values
```

Comment:
Nice! I'd inline this variable, too.

Reply:
IMO, this would make the code less readable (I understand this is completely subjective, so if you insist, I can do this). If I understand Rust compilation correctly, an additional local var leads to the same output at least with optimizations enabled.